packages/xo-web/src/xo-app/proxies/index.js

```diff
@@ -56,6 +56,13 @@ const ACTIONS = [
 ]
 
 const INDIVIDUAL_ACTIONS = [
+  {
+    collapsed: true,
+    disabled: ({ url }) => url === undefined,
```

Comment:
When can the URL be `undefined`? Is it common enough that we should change the tooltip to explain why the URL can't be copied?

/cc @julien-f https://github.com/vatesfr/xen-orchestra/pull/6287#issuecomment-1154819822

Reply:
The URL is built from the token (in database) and the VM IP address (fetched from the XAPI).

If the VM is not found or has no IP address, `url` is undefined.

Reply:
(Should usually not happen).

Reply:
Ok so "URL not found" or "Unavailable" should be enough when the URL is `undefined`.